RxSwift/Traits/PrimitiveSequence.swift

```diff
@@ -627,6 +627,20 @@ extension PrimitiveSequenceType where TraitType == MaybeTrait {
     }
 }
 
+extension PrimitiveSequence where Trait == CompletableTrait, Element == Never {
```

Comment:
Hi @hagmas ,

I think you need to make this into `PrimitiveSequence`. Unit tests are failing because of this IMHO.